packages/calcite-components/src/components/action/action.tsx

```diff
@@ -37,13 +37,13 @@ export class Action extends LitElement implements InteractiveComponent {
 
   //#region Private Properties
 
-  private guid = `calcite-action-${guid()}`;
```

Comment:
For IDs used to build other IDs, I think we can do the following:
 
```ts
// action
private guid = guid();
// ...
private buttonId = IDS.button(this.guid);
```
```ts
// action/resources.ts
export const IDS = {
  buttonId: (id: string) => `calcite-action-${id}-button`,
  indicatorId: (id: string) => `calcite-action-${id}-indicator`,
} as const;
// alternatively: `calcite-action` can be a const in this module and be used in the above string templates
```
 
It'd be good to have a consistent ID pattern, although this can be a follow-up.